sdk/typescript/src/cryptography/hash.ts

```diff
@@ -20,5 +20,5 @@ export function sha256Hash(typeTag: string, data: Uint8Array): string {
 
   hash.update(dataWithTag);
 
-  return Buffer.from(hash.hex(), 'hex').toString('base64');
+  return Buffer.from(hash.hex(), 'hex');
```

Comment:
When did we re-add buffer to the SDK? We should never use the native Buffer type.

Reply:
good catch, this was recently introduced, removed.